packages/sources/cfbenchmarks/src/endpoint/crypto.ts

```diff
@@ -30,6 +30,11 @@ export const inputParameters = new InputParameters(
       description: 'The symbol of the currency to convert to',
       required: false,
     },
+    adapterNameOverride: {
+      type: 'string',
+      description: 'Alternate adapter name to use for override and metrics',
```

Comment:
This won't take any effect if set be the user right ? Maybe we can clarify that in the description

Reply:
Good idea, user should not touch this